### PR TITLE
Add Fathom analytics + privacy policy

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

- Add Fathom script
- Add Fathom pageview tracking to Router
- Add PoC event for "Design your project" CTA
- Add privacy policy based on https://usefathom.com/docs/start/policy

## Screenshots or screen recordings

https://app.usefathom.com/share/eryrrjsv/juicebox.money

DM me for the password

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

File: src/Router.tsx [hunk 1]
```diff
@@ -37,63 +38,84 @@ const V2CreateRoute = () => (
...
   const isV2Enabled = featureFlagEnabled(FEATURE_FLAGS.ENABLE_V2)
 
   return (
-    <HashRouter>
-      <Switch>
-        <Route exact path="/">
-          <V1CurrencyProvider>
-            <Landing />
-          </V1CurrencyProvider>
+    <Switch>
```
Comment (aeolianeth) [inline src/Router.tsx:56]: Nothin changed here, just moved Switch to it's own component so we can use the usePageViews hook above it (but with HashRouter)

File: src/components/Landing/index.tsx [hunk 2]
```diff
@@ -177,7 +182,10 @@ export default function Landing() {
                       <Button
                         type="primary"
                         size="large"
-                        onClick={scrollToCreate}
+                        onClick={() => {
+                          fathom.trackGoal('IIYVJKNC', 0)
```
Comment (aeolianeth) [inline src/components/Landing/index.tsx:186]: Track clicks on this button, for example.

This is blocked by ad-blockers.

**Review by wraeth-eth — APPROVED** [review]
LFG

File: src/components/PrivacyPolicy.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import ExternalLink from './shared/ExternalLink'
+
+export default function PrivacyPolicy() {
```
Comment (aeolianeth) [inline src/components/PrivacyPolicy.tsx:3]: Source, https://usefathom.com/docs/start/policy

File: src/components/PrivacyPolicy.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <div style={{ padding: '40px', maxWidth: '1080px', margin: '0 auto' }}>
+      <h1>Privacy Policy</h1>
+      <p>
+        We want to process as little personal information as possible when you
```
Comment (benschac) [inline src/components/PrivacyPolicy.tsx:8]: quick q: Do we wanna translate this content?

File: src/components/Landing/index.tsx (hunk 2 above)
Comment (benschac) [reply src/components/Landing/index.tsx:186]: Would be dope to add some type safety one we get more `event-id`s but this looks solid.

**Review by benschac — APPROVED** [review]
LGTM 🍌

File: src/components/PrivacyPolicy.tsx (hunk 4 above)
Comment (johnnyd-eth) [reply src/components/PrivacyPolicy.tsx:8]: Because this is such a big piece of text and likely to be changed soon, I think it should be left unmarked

**Comment by johnnyd-eth** [comment]
Legend LGTM 🚀

**Review by johnnyd-eth — APPROVED** [review]

File: src/components/PrivacyPolicy.tsx (hunk 4 above)
Comment (aeolianeth) [reply src/components/PrivacyPolicy.tsx:8]: Gotta be careful translating stuff like this too. Might need a bit more of a stringent translation review process.

File: src/components/Landing/index.tsx (hunk 2 above)
Comment (aeolianeth) [reply src/components/Landing/index.tsx:186]: Yeah yeah for sure.

**Pull request merged**